packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -67,83 +67,44 @@ function Card(props: { propsToPass: any; type: string }) {
   }
 }
 
-export function CardContainer(props: CardContainerProps) {
+function _CardContainer(props: CardContainerProps) {
   const {
     isHeader = false,
     unitDescription,
     type,
-    isLoading,
     secondaryLabelLimit = 3,
     title,
-    initialLimit,
+    rowSet,
     ...rest
   } = props
-  const infiniteQueryContext = useInfiniteQueryContext()
   const { NoContentPlaceholder } = useQueryVisualizationContext()
-  const { appendNextPageToResults, hasNextPage } = infiniteQueryContext
-  const data = useAtomValue(tableQueryDataAtom)
+  const queryContext = useQueryContext()
+  const { queryMetadataQueryOptions } = queryContext
+  const { data: queryMetadata } = useSuspenseQuery(queryMetadataQueryOptions)
```

Comment:
using suspense to get the query metadata (everything that isn't in `QueryResultBundle.queryResults`)